auth/jwt/tokenizer.go

```diff
@@ -64,6 +73,13 @@ func (repo *tokenizer) Issue(key auth.Key) (string, error) {
 		Expiration(key.ExpiresAt)
 	builder.Claim(userField, key.User)
 	builder.Claim(domainField, key.Domain)
+	if key.OAuth.Provider == repo.google.Name() {
```

Comment:
Not related to this PR, but please rename the pointer receiver to `t` instead of `repo`. It's confusing.